webots/controllers/platoon_supervisor/platoon_supervisor.py

```diff
@@ -28,22 +84,19 @@ def main_loop():
 
     status = 0
```

Comment:
why is this the only place where the `status` variable is set? it is set to 0, then compared to 0 and the function always returns 0?